lua/starfall/libs_cl/render.lua

```diff
@@ -42,37 +42,73 @@ SF.RT_Material = RT_Material
 local playerFonts = SF.EntityTable("playerFonts")
 
 local defined_fonts = {
-	DebugFixed = true,
-	DebugFixedSmall = true,
-	Default = true,
-	Marlett = true,
-	Trebuchet18 = true,
-	Trebuchet24 = true,
-	HudHintTextLarge = true,
-	HudHintTextSmall = true,
-	CenterPrintText = true,
-	HudSelectionText = true,
-	CloseCaption_Normal = true,
-	CloseCaption_Bold = true,
-	CloseCaption_BoldItalic = true,
-	ChatFont = true,
-	TargetID = true,
-	TargetIDSmall = true,
-	HL2MPTypeDeath = true,
-	BudgetLabel = true,
-	HudNumbers = true,
-	DermaDefault = true,
-	DermaDefaultBold = true,
-	DermaLarge = true,
-	GModNotify = true,
-	ScoreboardDefault = true,
-	ScoreboardDefaultTitle = true,
-	GModToolName = true,
-	GModToolSubtitle = true,
-	GModToolHelp = true,
-	GModToolScreen = true,
-	ContentHeader = true,
-	GModWorldtip = true,
+	-- https://wiki.facepunch.com/gmod/Default_Fonts
```

Comment:
I'm not sure if all of these are defined fonts or if some are font file names. For example, Coolvetica is a font file, not an actually initialized font afaik.

Reply:
Need to check

Reply:
> You'd be better off undoing this change. It doesn't really add any benefits. Just makes things more confusing.

Agree to disagree. I updated the docs to provide more insight into why fonts may not be working, same citation is on the gmod wiki page:
https://wiki.facepunch.com/gmod/Finding_the_Font_Name#findthefontsname

On Windows/macOS, you are required to use font's **embedded** name, **not the font's filename.**
On Linux, it is the opposite, it is recommended to use the font filename (case-sensitive + .ttf)!